src/Forum/AuthenticationResponseFactory.php

```diff
@@ -104,7 +104,14 @@ private function getPayload(array $identification, array $suggestions, User $use
             $token = AuthToken::generate($identification);
             $token->save();
 
-            $payload = array_merge($identification, $suggestions, ['token' => $token->id]);
+            $payload = array_merge(
+                $identification,
+                $suggestions,
+                ['token' => $token->id],
+                // List of the fields that can't be edited during sign up
+                // Only includes attributes on the sign up form, otherwise this could leak private attribute names
+                ['identification_fields' => array_keys(array_only($identification, ['username', 'email']))]
```

Comment:
We're already sending `$identification` values through in the payload above (line 108), I don't think we need to worry about leaking private attribute names.

Please change the key to `identificationFields`.

Reply:
Didn't even notice it. It makes no sense to filter it then indeed :sweat_smile: 

My reasoning was to prevent a token like `twitter_id` (either its name or value) to be sent to the browser... But it's usually a public value anyway. I'll remove that